optimade/validator/validator.py

```diff
@@ -750,18 +754,21 @@ def _construct_single_property_filters(
 
                 reversed_response = reversed_response.json()
 
-                if (
-                    reversed_response["meta"]["data_returned"]
-                    != response["meta"]["data_returned"]
-                ):
-                    raise ResponseError(
-                        f"Query {query} did not work both ways around: {reversed_query}, "
-                        "returning different results each time."
-                    )
+                num_response = response["meta"].get("data_returned")
+                num_reversed_response = response["meta"].get("data_returned")
```

Comment:
I would probably do this all over. Unless you're sure you always have `"meta"` available?
```suggestion
                num_response = response.get("meta", {}).get("data_returned", data_returned)
                num_reversed_response = response.get("meta", {}).get("data_returned", data_returned)
```

Reply:
I don't really want to reintroduce all this flexibility again (as this has partly caused all this validator fragility from last week), I think I'd rather re-enable the strict deserialization now that implementations are getting closer and our error messages are getting better

Reply:
Okay, scratch that... doing the strict deserialization still fills the screen with pydantic validation errors for every non-OPT implementation I've tried, which perhaps isn't that useful...